core/jazz_cloud-logs-streamer/index.js

```diff
@@ -65,6 +65,7 @@ module.exports.handler = (input, context, cb) => {
 
                 if (failedItems && failedItems.length > 0) {
                     logger.error("Failed Items: " + JSON.stringify(failedItems, null, 2));
+                    return cb(JSON.stringify(errorHandler.throwInputValidationError("101", "Failed Items")));
```

Comment:
This isn’t a micro service so there is no point in returning a error code. Moreover line 71 will never run now